examples/IMAGE/mnist/mnist_config.yaml

```diff
@@ -0,0 +1,33 @@
+axon:
+  local_port: 8091
+  local_ip: 127.0.0.1
+  use_upnpc: False
+  max_workers: 10
+  max_gradients: 100
+receptor:
+  pass_gradients: true
+  timeout: 0.5
+  do_backoff: true
+  max_backoff: 100
+  key_dim: 100
+  stale_emit_filter: 10000
+  topk: 10
+metagraph:
+  chain_endpoint: feynman.akira.bittensor.com:9944 # Test network
+  stale_emit_filter: 10000
+session:
+  learning_rate: 0.001
+  momentum: 0.98
+  batch_size_train: 8
+  batch_size_test: 64
+  log_interval: 10
+  sync_interval: 10
+  name: mnist
+  record_log: true
+nucleus:
+  max_workers: 5
+  queue_maxsize: 10000
+  queue_timeout: 5
```

Comment:
same comment as CIFAR, take a look at `bittensor/synapses/ffnn.py` and add those args under `synapse:`